lms/product/d2l/_plugin/misc.py

```diff
@@ -3,10 +3,6 @@
 
 
 class D2LMiscPlugin(MiscPlugin):
-    # Deep linking in D2L implies creating a new assignment.
-    # Prompt for a title to set it for the new assignment.
-    deep_linking_prompt_for_title = True
```

Comment:
Already the default:


https://github.com/hypothesis/lms/blob/remove-redifintion/lms/product/plugin/misc.py#L26